cfme/tests/configure/test_logs.py

```diff
@@ -74,17 +75,48 @@ def test_provider_log_updated(appliance, provider, log_exists):
     Metadata:
         test_flag: log
     """
-    if log_exists:
-        log_before = appliance.ssh_client.run_command(
-            "md5sum /var/www/miq/vmdb/log/{}.log | awk '{{ print $1 }}'".format(
-                provider.log_name
-            )
-        ).output
+    assert log_exists, "Log file {}.log doesn't exist".format(provider.log_name)
+    log_before = appliance.ssh_client.run_command(
+        "md5sum /var/www/miq/vmdb/log/{}.log | awk '{{ print $1 }}'".format(
+            provider.log_name
+        )
+    ).output
 
-        wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600)
-        log_after = appliance.ssh_client.run_command(
-            "md5sum /var/www/miq/vmdb/log/{}.log | awk '{{ print $1 }}'".format(
-                provider.log_name
-            )
-        ).output
-        assert log_before != log_after, "Log hashes are the same"
+    wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600)
+    log_after = appliance.ssh_client.run_command(
+        "md5sum /var/www/miq/vmdb/log/{}.log | awk '{{ print $1 }}'".format(
+            provider.log_name
+        )
+    ).output
+    assert log_before != log_after, "Log hashes are the same"
+
+
+@pytest.mark.meta(blockers=[BZ(1633656,
+                               unblock=lambda provider: provider.one_of(AzureProvider, EC2Provider),
+                               forced_streams=["5.10", "upstream"])]
+                  )
+def test_provider_log_level(appliance, provider, log_exists):
+    """
+    Tests that log level in advanced settings affects log files
+
+    Steps:
+    1. Change log level to debug
+    2. Refresh provider
+    3. Check logs contain debug level
+    4. Reset level back
+
+    Metadata:
+        test_flag: log
+    """
+    assert log_exists, "Log file {}.log doesn't exist".format(provider.log_name)
+    log_level = appliance.server.advanced_settings['log']['level_{}'.format(provider.log_name)]
+    # set log level to debug
+    appliance.server.update_advanced_settings(
+        {'log': {'level_{}'.format(provider.log_name): 'debug'}})
+    wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600)
+    debug_in_logs = appliance.ssh_client.run_command(
+        "cat /var/www/miq/vmdb/log/{}.log | grep DEBUG".format(provider.log_name))
+    # set log level back
+    appliance.server.update_advanced_settings(
+        {'log': {'level_{}'.format(provider.log_name): log_level}})
+    assert debug_in_logs.success
```

Comment:
Seeing this failing in PRT against two providers in one version and several in the other - please comment on what's going on with this check? Does BZ 1633656 cover more providers than thought?

Reply:
@mshriver I updated the blocker as there's one more BZ, but for the remaining providers I have all the tests passed locally, however they're failing in PRT, can you please take a look why this can happen?